pkgs/dart_model/analysis_options.yaml

```diff
@@ -0,0 +1 @@
+include: package:lints/recommended.yaml
```

Comment:
lets go ahead and use the dart_flutter_team_lints?

Reply:
I'm fine either way - I just wanted to upgrade this from not using a config

Reply:
updated!